test/updaters/fixtures/README.md

```diff
@@ -0,0 +1,10 @@
+# Terraform Module
+
+## Usage
+
+```hcl
+module "vpc" {
```

Comment:
Is having a stanza like this fairly standard in the terraform community?

Reply:
Yes, it's a very common convention across many/most modules.